org.eclipse.winery.bpmn4tosca.converter.tobpel/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```

Comment:
I guess this file was created, therefore please remove 2017

Reply:
I added ADR-0025